openedx/core/djangoapps/ccxcon/__init__.py

```diff
@@ -6,4 +6,5 @@
 The ccxcon app needs to be placed in `openedx.core.djangoapps`
 because it will be used both in CMS and LMS.
 """
-import openedx.core.djangoapps.ccxcon.signals
+
+default_app_config = 'openedx.core.djangoapps.ccxcon.apps.CCXConnectorConfig'
```

Comment:
This change delays signal registration?  Is this related to moving around model imports, or does it comply with some other new django initialization requirement?

Reply:
It's a new best practice for adding configuration for a Django application - details are here:
https://docs.djangoproject.com/en/1.11/ref/applications/#configuring-applications